dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelector.js

```diff
@@ -305,22 +305,45 @@ dojo.declare(
             }
         },
 
+        _getSiteFolderFieldDefaultHTML: function (){
+            const defaultSiteFolderField = {
+                "fieldContentlet": "system_field",
+                "fieldFieldType": "host or folder",
+                "fieldName": "Site or Folder",
+                "fieldValues": "",
+                "fieldVelocityVarName": "siteOrFolder",
+            }
+
+            var htmlstr = "<dl class='vertical'>";
+            htmlstr += '<dt><label>' + this._fieldName(defaultSiteFolderField) + '</label></dt>';
+            htmlstr += '<dd>' + this._renderSearchField(defaultSiteFolderField) + '</dd>';
+            htmlstr += '</dl>';
+
+            return htmlstr;
+        },
+
         _fillFields: function (data) {
             this.currentStructureFields = data;
             this.search_fields_table.innerHTML = '';
+            this.site_folder_field_pop.innerHTML = '';
             var htmlstr = "<dl class='vertical'>";
             for (var i = 0; i < data.length; i++) {
                 var type = data[i]['fieldFieldType'];
-                if (type == 'category' || type == 'hidden') {
+                if (type == 'category' || type == 'hidden' || type == "host or folder") {
```

Comment:
same here I would change type to  "host_or_folder"

Reply:
`Field.FieldType.HOST_OR_FOLDER.toString()` can only be accessed in JSP or Java files